libs/core/src/lib/button/button.component.ts

```diff
@@ -1,21 +1,30 @@
 import { ChangeDetectionStrategy, Component, ElementRef, Input, ViewEncapsulation, OnInit } from '@angular/core';
 import { applyCssClass, CssClassBuilder } from '../utils/public_api';
 
-export type ButtonType = '' | 'standard' | 'positive' | 'negative' | 'attention' | 'half' | 'ghost' | 'transparent' | 'emphasized' | 'menu';
+export type ButtonType =
+    | ''
+    | 'standard'
+    | 'positive'
+    | 'negative'
+    | 'attention'
+    | 'half'
```

Comment:
Are we keeping the half button? @droshev

Reply:
I don't want to keep something that is not part of Fiori 3. At least not in the API

Reply:
I removed `half` from docs but for now I didn't remove `half` from component